### ISSUE TITLE
No bell in WSL?

### ISSUE THREAD

**Issue by Benosika** [Microsoft/BashOnWindows#1355]
I tried many things I found in Google to ring the good old Unix bell both in my Putty SSHed Ubuntu server 16.04 and then in my WSL 14.04...

I can run in both:

> echo -e "\07" && echo -e "\007"

Yet, no sound... Just take a pure WSL machine and run this, if you'll hear sound, than, I'll be surprised because I can't hear any sound and my speakers are fine and can play anything besides that at the same time... :\

So to reproduce --- Create a brand new WSL environment and run each of this bell commands to ring the bell.

Or else, there is no bell in WSL?

**Comment by aseering** [comment]
@Benosika -- interesting...  For what it's worth, `echo -e "\007"` causes a Windows system beep for me.

**Comment by Benosika** [comment]
Indeed... I personally have no knowledge in sound electronics && Sound development to explain this...

Maybe it's due to my hardware (I know that bell/beep sounds are not processed by the same units of typical system sounds and that's maybe where the issue is but I might be very wrong here).

I would very humbly suggest the WSL mods to consult the Win sound dev team on this...

**Comment by aseering** [comment]
Actually -- these days the system beep is no longer handled by dedicated hardware on most systems.  Instead, it is typically implemented by playing a ".wav" file through your system's regular sound system.  On Windows, you can configure this through the "Sound" Control Panel.  You might want to verify the system-beep configuration in that control panel.

If your system does use the dedicated PC Speaker directly, could you post more details about your system configuration?

**Comment by Benosika** [comment]
Indeed that was the problem! When I installed Win10 I muted all system sounds (By choosing "no sound scheme")... When I unmuted them I did heard this beep. I will now mute them back and will search for the exact beep sound.

About your second passage, my mainboard is Gigabye X58-USB3, it's a bit old (I think it's from 2009); I believe it does have a PC speaker but I normally use the native sound card.

I thank you dearly for your help! I will now report this as answer in Stackexchange. You not only helped me, but anyone else who will have this problem in this future. Thanks again !!!
